File: module7/code/back/springjdbcForm/src/main/java/com/edugroupe/springjdbcForm/metier/Produit.java
package com.edugroupe.springjdbcForm.metier;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @ToString
public class Produit {
	private int id;
	private String nom;
	private double prix;
	private double poids;

}

File: module10[mvc_angular]/front/boutiqueimagefront/src/app/metier/produit.ts
import { Categorie } from './categorie';
import { Image } from './image';

export class Produit {
    constructor(public id: number,
                public nom: string,
                public prix: number,
                public poids: number,
                public categorie?: Categorie,
                public images?: Image[]){}
}